demo/react/doc/product/components/editable-media.mdx

```diff
@@ -60,17 +60,70 @@ import { mdiDelete, mdiEye, mdiPencil } from 'LumX/icons';
     return(
         <>
             <EditableMedia
-                helper="Pick or upload a picture. It will show up in the preview of this content."
-                label="Featured Image"
-                placeholder="Add featured image"
+                placeholder="Add media"
                 image={image}
                 theme={theme}
                 onClick={handleClick}
                 theme={theme}
             >
                 {getControls()}
             </EditableMedia>
+        </>
+    );
+};
+```
+
+## Example : Profile picture
```

Comment:
This is a variant in the component's properties.
I'll put it that way:

## Variants

Editable medias come in two variants: _thumbnail_ and _avatar_